fodamy-baki/Scenes/RecipeDetails/RecipeDetailsViewModel.swift

```diff
@@ -79,6 +86,23 @@ extension RecipeDetailsViewModel {
         }
     }
     
+    func getRecipesDetailsComment() {
+        showLoading?()
+        dataProvider.request(for: GetRecipeCommentsRequest(recipeId: recipeId)) { [weak self] result in
+            guard let self = self else { return }
+            self.hideLoading?()
+            switch result {
+            case .success(let response):
+                let cellItems = response.data.prefix(3).map({ CommentCellModel(comment: $0) })
+                self.commentsCellItems.append(contentsOf: cellItems)
+                self.reloadCommentData?()
+                
```

Comment:
boşluk silinmeli, NOT: PR açmadan önce güzelce incele buradan eksiklieri kapatıp pr açarsan daha iyi olur